testutils/clusterlock/clusterlock_test.go

```diff
@@ -87,6 +87,15 @@ var _ = Describe("cluster lock test", func() {
 		Expect(lock2.AcquireLock()).NotTo(HaveOccurred())
 		Expect(lock2.ReleaseLock()).NotTo(HaveOccurred())
 	})
+
+	It("only the user holding the lock can release it", func() {
```

Comment:
What about a test that a different user trying to release does throw an error?

Reply:
That's what this test does